SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialReplicationGraph.cpp

```diff
@@ -90,6 +139,14 @@ void USpatialReplicationGraph::PostReplicateActors(UNetReplicationGraphConnectio
 	{
 		LoadBalancingHandler->ProcessMigrations();
 	}
+
+	USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(NetDriver);
```

Comment:
Nit: Guess this could be moved inside if.

Reply:
Moved